docker-compose.yml

```diff
@@ -33,7 +36,10 @@ services:
         target: /app
   applicants:
     build: ./bops-applicants
-    command: bash -c "bundle install && bundle exec rails s -p 3001 -b 0.0.0.0"
+    entrypoint: ./docker-entrypoint.sh
```

Comment:
Are you adding a PR for this file in  bops-applicant?

Reply:
https://github.com/unboxed/bops-applicants/pull/55